book/go/getting-started/setup.md

```diff
@@ -1,5 +1,5 @@
 # Setting Up Your Environment for Wallaroo
 
-It is currently possible to develop Wallaroo Go applications on Linux. This guide currently has installation instructions for Ubuntu Linux. It's assumed that if you are using a different Linux distribution then you are able to translate the Ubuntu instructions to your distribution of choice.
+It is currently possible to develop Wallaroo Go applications on 64-bit Linux. (NOTE: Wallaroo does not support 32-bit platforms.) This guide currently has installation instructions for Ubuntu Linux. It's assumed that if you are using a different Linux distribution then you are able to translate the Ubuntu instructions to your distribution of choice.
```

Comment:
I know this isn't part of your PR but I think this paragraph reads really weirdly.

>It is currently possible to develop Wallaroo Go applications on 64-bit Linux. (NOTE: Wallaroo does not support 32-bit platforms.) This guide currently has installation instructions for Ubuntu Linux. It's assumed that if you are using a different Linux distribution then you are able to translate the Ubuntu instructions to your distribution of choice.

to something like this maybe...

> Wallaroo Go applications only support 64-bit platforms. The guide below includes installation instructions for Ubuntu 16.04. It's assumed if you are using a different Linux distribution that you can translate the installation instructions to the Linux flavor of your choice. 

WDYT?

Reply:
Edited with a new commit